pages/common/jello.md

```diff
@@ -0,0 +1,32 @@
+# jello
+
+> A command-line JSON processor using python syntax.
+> More information: <https://github.com/kellyjonbrazil/jello>.
+
+- Output JSON or JSON Lines STDIN data, in pretty-print format:
+
+`cat {{file.json}} | jello`
+
+- Output a grepable schema from JSON or JSON Lines STDIN data:
```

Comment:
I think `grepable` is not a word. I can't really come up with an alternative, since I don't know what it's supposed to mean.

Reply:
Yeah it is a sort of made-up term. Another way to put it would be grep-able or greppable (https://github.com/tomnomnom/gron).

The idea is that you cannot easily use the `grep` command on JSON data, so this schema view allows you to use `grep` to find what you need more easily in the JSON.

Maybe something like: "Output a schema from JSON or JSON Lines data from stdin to stdout (useful for `grep`):

Reply:
I changed the language. Let me know if that doesn't work.